pyblazing/pyblazing/apiv2/context.py

```diff
@@ -1504,11 +1507,19 @@ def sql(self, sql, table_list=[], algebra=None, use_execution_graph=True):
             elif(new_tables[table].fileType == DataType.CUDF or new_tables[table].fileType == DataType.ARROW):
                 currentTableNodes = []
                 for node in self.nodes:
+                    inputdf = new_tables[table].input
+
+                    usage = inputdf.memory_usage(deep=True).sum()
+                    if usage > 1073741824: #1GB
```

Comment:
No deberiamos hacer esto. Si un dataframe entra y es grande, no importa, dejalo asi. Si lo partes en pedazos, entonces lo que estas haciendo es duplicando la memoria que se esta ocupando en el GPU, porque el input original no lo puedes modificar

Reply:
Got it, the code was removed.